frontend_tests/node_tests/stream_data.js

```diff
@@ -201,6 +201,11 @@ zrequire('stream_data');
     stream_data.update_subscribers_count(sub);
     assert.equal(sub.subscriber_count, 0);
 
+    // verify that checking subscription with bad email is a noop
+    global.blueslip.error = function () {};
```

Comment:
A lot of these tests were added over a year ago, so now we usually practice more flushed-out testing practices. It may seem like overkill here, but it's good practice and will come into good play when you try to test harder/larger functions.

For example, for this `blueslip.error`, which is called in [`people.js` L93](https://github.com/zulip/zulip/blob/master/static/js/people.js#L93), we can actually have the function itself `assert` something. So, looking at the code in `people.js`, we see that `blueslip.error` is given an `error_msg`. We can then change our test code to be:

```js
global.blueslip.error = function (error_msg) {
    assert.equal(error_msg, "whatever the error message is");
};
```

Similarly, we can deal with the `blueslip.warn` in the same way:

```js
global.blueslip.warn = function (msg) {
    assert.equal(msg, "Bad email passed to user_is_subscribed: " + user_email);
};
```

If you still don't understand, just ask on #GCI help or something and ping me!